prqlc/bindings/js/package.json

```diff
@@ -12,8 +12,8 @@
   ],
   "license": "Apache-2.0",
   "author": "PRQL team",
-  "main": "dist/node/prql_js.js",
-  "name": "prql-js",
+  "main": "dist/node/prqlc_js.js",
+  "name": "prqlc-js",
```

Comment:
> But I'd like to find a way of getting the npm upload to be prqlc

I suppose this line should be `"name": "prqlc"`
<https://docs.npmjs.com/cli/v6/configuring-npm/package-json#name>

Reply:
Oh nice, thanks. Do you think this takes care of:

> But I'd like to find a way of getting the npm upload to be `prqlc`, rather than `prqlc-js`. We manage this in python...

Reply:
Updated — let's see!